mne/io/base.py

```diff
@@ -1450,6 +1453,48 @@ def save(self, fname, picks=None, tmin=0, tmax=None, buffer_size_sec=None,
                    start, stop, buffer_size, projector, drop_small_buffer,
                    split_size, split_naming, 0, None, overwrite)
 
+    @verbose
+    def export(self, fname, fmt='auto', verbose=None):
+        """Export Raw to external formats.
+
+        Supported formats: EEGLAB (set, uses :mod:`eeglabio`)
+        %(export_warning)s
+        %(export_params_base)s
+            Valid options are ``'auto'`` for auto inferred and
+            ``'set'`` for EEGLAB.
```

Comment:
shouldn't we maybe call the format "eeglab" instead of "set"? WDYT?

Reply:
I think eeglab probably will make more sense once there are more formats. I used set just because it was easier to get the format automatically.

Reply:
I just updated the supported format list to a dict (e.g. 'eeglab': 'set').